### ISSUE TITLE
Реализация сетки bootstap по БЭМ

### ISSUE THREAD

**Issue by Degtyarev-vg** [bem/bem-forum-content-ru#1126]
Хочется использовать сетку bootstrap в БЭМ. Для этого переделал сетку bootstrap по методологии БЭМ. Саму методологию БЭМ использую только в css. Вроде никаких нарушений нет при внедрении сетки, но т.к. этой методологией только начал пользоваться, хочется услышать мнение профессионалов, может есть какие-либо замечания.
Сразу скажу, что библиотеку bem components использовать не желаю, т.к. мне проще верстать элементы самому. Сетку bem grid использовать не приходилось, т.к. уж очень много заморочек при ее интеграции в проект.
А вот сам код html:

    <div class="container">
		<div class="container__row">
			<div class="container__col container__col_md_4 container__col_xs_6">Блок 1</div>
			<div class="container__col container__col_md_4 container__col_xs_6">Блок 2</div>
			<div class="container__col container__col_md_4 container__col_xs_6">Блок 3</div>
		</div>
	</div>

Т.е. блок container так и остался, у него есть элемент container__row, и обязательный элемент container__col, т.к. модификаторы container__col_**_* нельзя использовать самостоятельно, т.к. с точки зрения БЭМ-методологии модификатор не может использоваться в отрыве от модифицируемого блока или элемента. 

Очень надеюсь на ответ. Спасибо!

**Comment by awinogradov** [comment]
@Degtyarev-vg http://theprotein.github.io/sharps/gh-pages/index.html не подошел почему?

**Comment by Degtyarev-vg** [comment]
Сетки основанные на flex конечно хороши, но они не работают в safari 5.1.7, а также в ie 9 версии, а у меня постоянный хороший заказчик требует совместимости в этих браузерах, поэтому и пришлось самому делать сетку. Но так как я только начинаю пользоваться технологией БЭМ хотелось бы узнать, не нарушает какие-либо правила методологии данная сетка.

**Comment by awinogradov** [comment]
Секта работает в IE9+ с фодбэком на calc. Но если желание все еще есть, то мне кажется, что более правильно было бы так:
```
.row
  .row__col.row__col_md_4
  .row__col.row__col_md_8
```

**Comment by amel-true** [comment]
Решал такую же задачу https://github.com/amel-true/am-grid

**Comment by Degtyarev-vg** [comment]
Круто! А у тебя нет исходников этой сетки в sass?

**Comment by Degtyarev-vg** [comment]
И там не хватает файла variables, в котором ты задаешь все переменные, хотя в файле readme написано, что он должен присутствовать.

**Comment by amel-true** [comment]
Нет, сетка реализована для Stylus. Файл настроект на месте https://github.com/amel-true/am-grid/blob/master/_type/am-grid_type_example.styl - подключается как зависимость. Задача была иметь разные сетки в проекте.

**Comment by nucknine** [comment]
@amel-true не понял как подключить сетку. 
Собрал все стили в один файл https://codepen.io/nucknine/pen/aWPPOw am-grid__col_md_ не работает.
Что я не так делаю?
